vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/EditColumnConfigurator.java

```diff
@@ -61,6 +66,16 @@ public EditColumnConfigurator(EditColumn<T> column) {
         return getColumn();
     }
 
+    private <V> Column<T> configureColumn(ItemUpdater<T, String> itemUpdater,
+                                      EditorType type, List<String> options, HasValueAndElement<?, V> editorField) {
+        column.setEditorType(type);
+        column.setItemUpdater(itemUpdater);
+        column.setOptions(options);
+        column.setEditorField((Component) editorField);
```

Comment:
Avoid casting: use `AbstractField` as the type instead of `HasValueAndElement`

Reply:
Replaced.